.github/workflows/cypress-ci.yaml

```diff
@@ -0,0 +1,44 @@
+name: Cypress Tests
+on: [push]
+  
+jobs:
+  cypress-run:
+    runs-on: ubuntu-22.04
+    steps:
+     - name: Checkout
+       uses: actions/checkout@v4
+     - name: Setup Node.js
+       uses: actions/setup-node@v4
+       with:
+         node-version: '20'
+         
+     - name: cache
+       uses: actions/cache@v4
+       with:
+            path: "tests/functional"
+            key: node-modules-${{ hashFiles('tests/functional/package.json') }}
+
+     - name: cypress install
+       uses: cypress-io/github-action@v6
+       with:
+          
+          working-directory: '${{ github.workspace }}/tests/functional'
+       env:
+          CYPRESS_keycloakUsername: ${{secrets.keycloakUsername}}
+          CYPRESS_keycloakPassword: ${{secrets.keycloakPassword}}
+          CYPRESS_depEnv: ${{secrets.depEnv}}
```

Comment:
According to the documentation this sets the path in the URL for example:
`https://chefs-dev.apps.silver.devops.gov.bc.ca/app` or `https://chefs-dev.apps.silver.devops.gov.bc.ca/pr-X`.

This seems like it would become very tedious or the user would have issues every time they forget to change it back to the app path. I think it would be better to set the default value as app and only change it to the pr-X if the developer supplies that value.

Take a look at our deploy script to see how we set the default values and then update them if they exist, and then use them further down the script. Look at the `ROUTE_PATH` variable specifically. It's exactly what you're looking for.
https://github.com/bcgov/common-hosted-form-service/blob/main/.github/workflows/.deploy.yaml#L19